allvers/1999-01-28-GPL-2.0-from-gnu.org.html.notes

```diff
@@ -0,0 +1,8 @@
+Date: 1999-01-28 or earlier
```

Comment:
The previous version was last captured in December 1998. The correct range when it was changed is between December 7, 1998 and January 28, 1999

Reply:
Updated in 52312e28ad0f3ae56593d2589fe584d0e67b3bb9